### DISCUSSION TITLE
wx5500

### DISCUSSION THREAD

**Discussion by xone1956** [khoih-prog/AsyncWebServer_STM32#8] [General]
Hallo,
does the asyncWebServer  also work with wx5500 ?

**Comment by khoih-prog** [comment]
> does the asyncWebServer also work with wx5500 ?

No. Only built-in **LAN8742A** or some F4/F7  using **LAN8720** with STM32 core v2.2.0-

Check [**Currently supported Boards**](https://github.com/khoih-prog/AsyncWebServer_STM32#currently-supported-boards) for more details.

**Comment by khoih-prog** [comment]
The only board and library currently supports Async for **W5x00** is **ESP8266** using [**AsyncWebServer_Ethernet** library](https://github.com/khoih-prog/AsyncWebServer_Ethernet)

**Comment by xone1956** [comment]
too bad, i had hope, OK
Thanks
